docs/development/releasing.rst

```diff
@@ -175,26 +171,28 @@ packages that use the full bugfix/maintenance branch approach.)
 
 #. Push up all these changes to the `astropy core repository`_::
 
-      $ git push --tags upstream v<version>
+      $ git push --tags upstream v<version branch>.x
```

Comment:
This is a recurring issue, for some reason we keep pushing a stable tag accidentally, but normally stable is a branch. The solution is just to remove the stable tag after pushing. I don't think anyone ever figured out where it comes from.